src/services/projects.js

```diff
@@ -729,6 +725,14 @@ module.exports = class ProjectsHelper {
 			}
 
 			await resourceQueries.updateOne({ id: projectData.id }, resourcesUpdate)
+			//add user action
+			eventEmitter.emit(common.EVENT_ADD_USER_ACTION, {
```

Comment:
@priyanka-TL why is this added here and not via hook ?

Reply:
@aks30, as we've discussed before, once the user submits the resource for a second time or more, we can't identify that through the hook. The first submission sets the resource status to 'submitted,' but for subsequent submissions, it changes to 'in_review'.